daemon/info.go

```diff
@@ -75,7 +75,7 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		DriverStatus:       daemon.layerStore.DriverStatus(),
 		Plugins:            daemon.showPluginsInfo(),
 		IPv4Forwarding:     !sysInfo.IPv4ForwardingDisabled,
-		BridgeNfIptables:   !sysInfo.BridgeNfCallIptablesDisabled,
+		BridgeNfIptables:   !sysInfo.BridgeNfCallIPtablesDisabled,
```

Comment:
Should it be `IPv4` / `IPv6` in all of them? (With a `v` in the middle)? - Just a suggestion / possible alternstive